workflows/photovoltaic.md

```diff
@@ -15,7 +15,15 @@ Community Photovoltaic features can be added to the GeoJSON Feature File as a di
+Ground Mount Photovoltaic features can also be added to the GeoJSON Feature File as a district system feature of type "Ground Mount Photovoltaic". A ground mount PV feature should have an `associated_building_id` element specified, with the ID of a building feature in the Feature File. Grount Mount photovoltaic systems are assumed to be "behind the meter" of the utility meter for the associated building feature. View the [district system schema](https://docs.urbanopt.net/urbanopt-geojson-gem/schemas/district-system-properties.html) for more details. 
+
+The REopt optimization post-processors will use this information in their PV capacity calculations provided that there is a PV specification with location equal to `ground` in the REopt [assumptions file](#assumptions-files) for the building feature with associated ground mount PV. (See the `multi-pv` assumptions file in the `reopt` folder of your project directory for an example.) A `gcr` field representing the ground coverage ratio, or the percentage of the area given by the ground mount feature area that will be covered by PV, is also found in the REopt [assumptions file](#assumptions-files). It is defaulted to 0.99 but can be adjusted to represent the PV coverage of the ground mount feature. More information on REopt assumptions files can be found in the [Assumptions Files](#assumptions-files) section below.
+
+Alternatively, ground mount photovoltaics can be considered in the REopt feature optimization without defining district system features and associating them with a building in the GeoJSON file. If no features are defined in the GeoJSON file, REopt return the optimal amount of ground mount PV without any area limitations for each feature that was run with an [assumptions file](#assumptions-files) that contained a PV definition with location = `ground`. More realistically, there will be area limitations and, therefore, district systems defined in the GeoJSON files for those building features where ground mount PV should be considered.  An additional refinement would be to specify different assumptions files for features with ground mount PV and those without. Feature-by-feature assumptions file assignment is done in the scenario CSV file for a particular [reopt-enabled](../getting_started/getting_started.md#step2) scenario. 
```

Comment:
Can we say return the optimal amount of ground mount PV **with an upper limit of area equivalent to the site are**a for each feature that was run with an [assumptions file](#assumptions-files) that contained a PV definition with location = `ground`.  More realistically, there will **be a more specific area limitation based on the area available for Ground Mount PV associated with the Feature** and, therefore, ....

Reply:
thanks! this should be fixed